code/controllers/subsystem/redis.dm

```diff
@@ -75,7 +85,15 @@ SUBSYSTEM_DEF(redis)
 
 	for(var/channel in usable_data)
 		if(channel == RUSTG_REDIS_ERROR_CHANNEL)
-			message_admins("Redis error: [usable_data[channel]] | Please inform the server host.") // uh oh
+			var/redis_error_data = usable_data[channel]
+			var/error_str
+			if(islist(redis_error_data))
+				error_str = json_encode(redis_error_data)
+			else
+				error_str = redis_error_data
+
+			message_admins("Redis error: [error_str] | Please inform the server host.") // uh oh
+			log_game("Redis error: [error_str]")
```

Comment:
Feel like this should be something other than `log_game`.

Maybe `log_misc` or `log_world` or something.